### PR TITLE
Home graph

### PR DESCRIPTION
**Pull request by lukeschunk**
### Hello Ryan :neckbeard:

This graph isn't totally done but I want to push it now, work on the photo collage, then dial it in when that's done.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
 	<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7/angular.js"></script>
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.js"></script>
 	<script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
+	<script src=""></script>
```
Comment (lukeschunk) [inline public/index.html]: don't know why i have this blank script - i will fix

File: main/client/features/home-chart/home-chart-dir.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
+angular.module('app').directive('linearChart', function($parse, $window) {
+
+    return {
+        restrict: 'EA',
```
Comment (r-walsh) [inline main/client/features/home-chart/home-chart-dir.js]: comma first

File: main/client/styles/login.styl [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+	border-radius 20px
+	padding-top 60px
+	padding-left 20px
+
+	a
+		text-decoration none
+		font-size 1.2em
+		font-family sans-serif
+		text-align center
+		color white
```
Comment (lukeschunk) [inline main/client/styles/login.styl]: this stylesheet can be deleted

File: main/client/features/home-chart/home-chart-dir.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                } else {
+                    return 'rgba(0, 114, 188, .95)'
+                }
+            }
+            // Create Circles
+            svg.selectAll("circle")
+                .data(dataset)
+                .enter()
+                .append("circle") // Add circle svg
+                .attr("cx", function(d) {
```
Comment (r-walsh) [inline main/client/features/home-chart/home-chart-dir.js]: Not mandatory, but try to be consistent in ES6 usage. Try to not swap between fat arrow/standard declaration regularly in the same file unless needed due to scoping.

Comment (lukeschunk) [inline main/client/features/home-chart/home-chart-dir.js]: Okay - will fix

File: public/index.html (hunk 1 above)
Comment (r-walsh) [inline public/index.html]: What is this?

**Pull request merged**